launcher

```diff
@@ -170,9 +170,13 @@ start() {
     set_existing_container
     if [[ $existing != "" ]]; then
         loginfo "starting up existing container"
+
+        local attach_on_start
+        [[ "${SUPERVISED}" = "true" ]] && attach_on_start="-a"
+        
         (
             set -x
-            docker start seafile
+            docker start $attach_on_start seafile
```

Comment:
The handling of `SUPERVISED` is already there, check https://github.com/haiwen/seafile-docker/blob/f54fbbb/launcher#L185 . 

Also this `if` block is only executed for the first time where the seafile container is not created yet, i.e. when the user runs `./launch bootstrap` . 

Thus,  I don't think it's necessary to handle `SUPERVISED` here.